website/static/img/databricks_tutorial/images/.gitignore

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
@amychen1776 This one can also be removed!

Reply:
Got it!